code/object/object_flags.h

```diff
@@ -27,6 +27,7 @@ namespace Object {
 		Marked,					// Object is marked (Fred).  Can be reused in FreeSpace for anything that won't be used by Fred.
 		Temp_marked,			// Temporarily marked (Fred).
 		Hidden,					// Object is hidden (not shown) and can't be manipulated
+		Collides_with_parent,	// Asteroth
```

Comment:
My only issue with this is that this needs a comment that it should only be set for weapons.   My initial reading of the changes to the collision code made me think that there was a mistake because a ship with the flag would have all of its weapons potentially collide with it.

Reply:
This was my fear as well, but I never observed it during my testing ¯\_(ツ)_/¯

Perhaps with different/more elaborate fire point set ups this could happen, but the sad fact is that if it happens, it would be very difficult to elegantly solve in the code, and by far the simplest solution would be to simply slightly move the firepoints away.